stable/node-local-dns/templates/daemonset.yaml

```diff
@@ -61,8 +61,13 @@ spec:
         resources: {{- toYaml . | nindent 10 }}
         {{- end }}
         args:
+        {{- if .Values.config.setupIptables }}
```

Comment:
Here you're essentially wrapping the existing args in a boolean that is currently `false` by default. So this change would adjust every user's args now to `-localip {{ .Values.config.localDns }}` unless they have already set `.Values.config.setupIptables=false`.

Do you that's desired?

Reply:
Thanks for fast reply.

My colleague @kkonnov mistakenly changed defaults in this [commit](https://github.com/deliveryhero/helm-charts/pull/503/commits/3b97be911095083e8e94e50d7a63e211639ca172).
For now I corrected defaults to return backward compatibility. 
JFYI: we are using specific configuration with cilium [local-policy-redirect](https://github.com/cilium/cilium/tree/main/examples/kubernetes-local-redirect) and don't iptables at all